idd/Energy+.idd.in

```diff
@@ -44025,7 +44025,13 @@ ZoneHVAC:LowTemperatureRadiant:VariableFlow,
         \key SurfaceFaceTemperature
         \key SurfaceInteriorTemperature
         \default MeanAirTemperature
-   A6 , \field Heating Design Capacity Method
+   A6,  \field Setpoint Control Type
```

Comment:
Do you need to alter the `\min-fields` for this object because of your new field?

Reply:
Fixed this--you are correct there were two of these that needed to be updated.

Reply:
:+1: